hw/ip_templates/rv_plic/fpv/tb/assertions.tcl

```diff
@@ -0,0 +1,15 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# To support that the precondition related to assertion Onehot0Check_A can never happen i.e.
+# !$onehot0(oh_i), an assertion to make sure that oh_i is always onehot or zero.
+assert -name InpAlwaysOnehot0_A {$onehot0(dut.u_reg.u_prim_reg_we_check.u_prim_onehot_check.oh_i)}
```

Comment:
These assertions look good, but can they be moved to the same file as the waiver itself? I think that will make the relationship a bit more obvious.